test/tests/functional/pbs_hook_execjob_prologue.py

```diff
@@ -223,7 +223,7 @@ def test_prologue_hook_fail_action_non_mom_hook(self):
         """
         Test that when fail action is set to anything else than 'None'
         and the mom hook is not one of the following- execjob_begin,
-        exechost_startup, execjob_prologue error msg is displayed
+        exechost_startup, execjob_prologue, error msg is displayed
```

Comment:
How about: "Test when the fail action is set to anything other than 'None' on a mom_hook, and that mom_hook is not either execjob_begin, exechost_startup, or execjob_prologue an error message is displayed"